lib/experimental/fonts/font-library/font-library.php

```diff
@@ -141,8 +128,11 @@ function wp_unregister_font_collection( $collection_id ) {
 	}
 }
 
-// TODO: update to production font collection URL.
-wp_register_font_collection_from_json( 'https://raw.githubusercontent.com/WordPress/google-fonts-to-wordpress-collection/01aa57731575bd13f9db8d86ab80a2d74e28a1ac/releases/gutenberg-17.6/collections/google-fonts-with-preview.json' );
+function gutenberg_register_font_collections() {
+	// TODO: update to production font collection URL.
+	wp_register_font_collection( 'google-fonts', 'https://raw.githubusercontent.com/WordPress/google-fonts-to-wordpress-collection/01aa57731575bd13f9db8d86ab80a2d74e28a1ac/releases/gutenberg-17.6/collections/google-fonts-with-preview.json' );
+}
+add_action( 'init', 'gutenberg_register_font_collections' );
```

Comment:
Ideally I think we'd register the collection only in wp-admin and for REST API requests, but I don't know of a good hook that works for both of those.

Reply:
We could use admin_init and rest_api_init with the same callback in that case.

Reply:
That would work well here for Gutenberg, but thinking it about it more, using 2 hooks seems like an awkward recommendation generally for font collection registration.

Reply:
Makes sense.